src/test/helper.ts

```diff
@@ -26,3 +27,7 @@ export const getDocumentPath = (p: string) => {
 export const getDocumentUri = (p: string) => {
   return vscode.Uri.file(getDocumentPath(p));
 };
+
+export const isWindows = () => {
+  return platform() === "win32";
+};
```

Comment:
I think we can remove this function as it's not actually being used in `server.ts`.

Reply:
I created the function for the tester `src\test\utils.test.ts`. Should I move it to the tester?

Reply:
Ah, didn't realize it's defined as a test helper function. It's fine then.